components/core/SlateLayout.js

```diff
@@ -692,13 +713,27 @@ export class SlateLayout extends React.Component {
       e.preventDefault();
       e.stopPropagation();
       this._handleSaveLayout();
+    } else if (
+      (this.keysPressed["Control"] || this.keysPressed["Meta"]) &&
+      prevValue !== this.keysPressed[e.key]
+    ) {
+      e.preventDefault();
+      e.stopPropagation();
+      this._handleHelpKeybind();
+      // either pressed key or unpressed, should make an event handler that gives true or
+      // false for key press and then use if for the rendering of the buttons?
     }
   };
 
   _handleKeyUp = (e) => {
+    this.setState({ keyboardTooltip: false });
     this.keysPressed = {};
```

Comment:
Do a `this._keysPressed` to let people know this is a private variable not meant to accessed out of the class :) good semantics.